src/main/java/mate/academy/Main.java

```diff
@@ -1,11 +1,13 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
 import mate.academy.model.Movie;
 import mate.academy.service.MovieService;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Injector injector = Injector.getInstance("mate.academy");
```

Comment:
Ensure that the `Injector` class is correctly implemented and configured to handle dependency injection properly. This is crucial for obtaining instances of services like `MovieService`.